zeek/assets/dashboards/corelight_suricata.json

```
@@ -0,0 +1,531 @@
+{
+    "title": "Corelight Suricata",
+    "description": "-   This dashboard will give the signature-based alerts.",
```

Please fact check my suggestion here. 
```suggestion
    "description": "This dashboard gives an overview of the Corelight Suricata signature-based alerts.",
```

---

Done